docs/self-managed/zeebe-deployment/operations/update-zeebe.md

```diff
@@ -194,6 +198,10 @@ This should resolve automatically once the broker is updated.
 
 If this persists, you may want to [force the update](#rolling-update-is-not-completing).
 
+:::note
+In this specific case, a rollback is possible because users may restart Zeebe with the "skipped" minor version. [Backup and restore](/self-managed/operational-guides/backup-restore/backup-and-restore.md) is possible, but results in partial data loss of everything following the backup. Therefore, a rollback is preferable in this use case.
```

Comment:
Similar to above, is it beneficial to combine this section?
```suggestion

If this persists, you can [force the update](#rolling-update-is-not-completing). Alternatively, it is possible to restart Zeebe with the "skipped" minor version.  Note that version rollbacks are not supported in most other instances.
```

Reply:
Adjusted in latest commit.